rclcpp/src/rclcpp/experimental/timers_manager.cpp

```diff
@@ -28,9 +28,9 @@ using rclcpp::experimental::TimersManager;
 TimersManager::TimersManager(
   std::shared_ptr<rclcpp::Context> context,
   std::function<void(const rclcpp::TimerBase *)> on_ready_callback)
+: context_(context),
+  on_ready_callback_(on_ready_callback)
```

Comment:
i think we should remove the following?

https://github.com/ros2/rclcpp/blob/5e152d77d8144c074894de9a5bc7025aac7f5813/rclcpp/include/rclcpp/experimental/timers_manager.hpp#L530

Reply:
Sure.  I don't think it matters too much either way, but we are sure it is always going to get set on construction via the initialization variables, so we can remove it.  I've done that in 37edae0.